src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Publish.targets

```diff
@@ -36,6 +36,10 @@ Copyright (c) .NET Foundation. All rights reserved.
     </ResolvedFileToPublish>
   </ItemDefinitionGroup>
 
+  <ItemGroup>
```

Comment:
I'm not crazy about putting this in between the "Publish" comment above and the "Publish" target below.  I see there is already "stuff" in between them, but adding to this makes it messy, IMO.